scipy/stats/tests/test_stats.py

```diff
@@ -3885,6 +3886,17 @@ def test_gh11184_bigger(self):
         self._testOne(x, y, 'greater', 0.10597913208679133, 2.7947433906389253e-41, mode='asymp')
         self._testOne(x, y, 'less', 0.09658002199780022, 2.7947433906389253e-41, mode='asymp')
 
+    @pytest.mark.slow
+    def test_gh12999(self):
+        for x in range(1000, 12000, 1000):
```

Comment:
This looks like a typo, did you mean `10_000` or `100_000` for the last value?

Reply:
This is intended to run from from `1_000` to `12_000` (exclusive, so `11_000`) in steps of `1_000`.

Reply:
never mind, sorry I misread that